lib/commands/instrument.js

```diff
@@ -51,14 +56,37 @@ exports.builder = function (yargs) {
       type: 'boolean',
       description: 'should nyc exit when an instrumentation failure occurs?'
     })
+    .option('exclude', {
+      alias: 'x',
+      default: testExclude.defaultExclude,
+      describe: 'a list of specific files and directories that should not be instrumented, glob patterns are supported'
+    })
+    .option('include', {
+      alias: 'n',
+      default: [],
+      describe: 'a list of specific files that should be instrumented, glob patterns are supported'
+    })
+    .option('cwd', {
+      describe: 'working directory used when resolving paths',
+      default: cwd
+    })
+    .option('es-modules', {
+      default: true,
+      type: 'boolean',
+      describe: 'tell the instrumenter to treat files as ES Modules'
+    })
+    .option('clean', {
+      describe: 'should the output folder be cleaned before instrumenting files?',
+      default: true,
```

Comment:
Hi Corey, I've tried to write this a few times but it was getting a little wordy. So quite simply, I don't feel there's much point in releasing this without the exclude, include and includeNegated changes.  So if it needs to wait for a major release then that's what should happen.  That way it can be part of a comprehensive effort to address missing functionality in the instrument command and be a PR that you're satisfied doesn't have any issues.

Reply:
If we do wait for the next major release, might be worth revisiting some of the exclude logic in general; it's a real pain in the butt when combined with source-maps -- I'd argue it might be worth expanding on the test suite to cover all these weird edge-cases, so that it's easier to add new features like this in the future, without bumping into foot guns (i.e., me and @coreyfarrell not being sure how safe the change is).